servicemesh-install/topics/servicemesh-istio-comparison.adoc

```diff
@@ -0,0 +1,45 @@
+[[servicemesh-istio-comparison]]
+= Comparing {ProductName} and upstream Istio community installations
+An installation of {ProductName} differs from upstream Istio community installations in multiple ways. The modifications to {ProductName} are sometimes necessary to resolve issues, provide additional features, or to handle differences when deploying on OpenShift.
+
+The current release of {ProductName} differs from the current upstream Istio community release in the following ways:
+
+== Automatic injection
+The upstream Istio community installation automatically injects the sidecar to namespaces you have labeled.
+
+{ProductName} does not automatically inject the sidecar to any namespaces, but requires you to specify sidecar injection in the https://docs.openshift.com/container-platform/3.11/servicemesh-install/servicemesh-install.html#updating-master-configuration[master configuration].
+
```

Comment:
The automatic injection should refer to the annotation rather than configuration of the webhooks, IIRC we have a section in the documentation which has the text so we could refer to that section if necessary.